### PR TITLE
=build Fix commandAlias for fixall and sortImports

### PR DESCRIPTION
**Pull request by He-Pin**
refs https://github.com/akka/akka/issues/28982

### CODE REVIEW

File: akka-docs/src/test/scala-jdk9-only/docs/stream/operators/source/AsSubscriber.scala [hunk 1]
```diff
@@ -5,12 +5,12 @@
 package docs.stream.operators.source;
```
Comment (patriknw) [inline akka-docs/src/test/scala-jdk9-only/docs/stream/operators/source/AsSubscriber.scala:5]: another docs package that shouldn't be touched?

File: build.sbt [hunk 2]
```diff
@@ -15,8 +15,7 @@ addCommandAlias(
 
 addCommandAlias(
   name = "sortImports",
-  value = ";scalafixEnable;compile:scalafix SortImports;test:scalafix SortImports;multi-jvm:scalafix SortImports;" +
-    "CompileJdk9:scalafix SortImports;TestJdk9:scalafix SortImports;scalafmtAll;test:compile;multi-jvm:compile;reload")
+  value = ";scalafixEnable;compile:scalafix SortImports;test:scalafix SortImports;scalafmtAll")
```
Comment (patriknw) [inline build.sbt:18]: why is scalafmtAll needed in the alias? does the sorting change the format of the imports? Is that for the grouped imports?

Comment (hepin1989) [reply build.sbt:18]: it sometimes change to `{ A }` to `{A}`

File: akka-docs/src/test/scala-jdk9-only/docs/stream/operators/source/AsSubscriber.scala (hunk 1 above)
Comment (hepin1989) [reply akka-docs/src/test/scala-jdk9-only/docs/stream/operators/source/AsSubscriber.scala:5]: Will check then.

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by hepin1989** [comment]
The real problem was caused by the file to package conversion

**Comment by hepin1989** [comment]
Includes https://github.com/akka/akka/pull/29006

File: project/ScalaFixExtraRulesPlugin.scala [hunk 3]
```diff
@@ -15,6 +15,6 @@ object ScalaFixExtraRulesPlugin extends AutoPlugin with ScalafixSupport{
   import sbt._
   import scalafix.sbt.ScalafixPlugin.autoImport.scalafixDependencies
   override def projectSettings: Seq[Def.Setting[_]] = super.projectSettings ++ {
-    scalafixDependencies in ThisBuild += "com.nequissimus" %% "sort-imports" % "0.4.0"
+    scalafixDependencies in ThisBuild += "com.nequissimus" %% "sort-imports" % "0.5.0"
```
Comment (hepin1989) [inline project/ScalaFixExtraRulesPlugin.scala:18]: update to new version

File: .scalafix.conf [hunk 4]
```diff
@@ -31,14 +31,15 @@ ignored-files = [
...
+  "docs",
   "doc",
   "jdoc"
 ]
 
 //sort imports, see https://github.com/NeQuissimus/sort-imports
 SortImports.asciiSort = false
 SortImports.blocks = [
-  "java.",
+  "re:javax?\\.",
```
Comment (hepin1989) [inline .scalafix.conf:42]: with javax

**Review by raboof — APPROVED** [review]

**Comment by hepin1989** [comment]
seems like I forget a rebase...

**Pull request merged**